app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -21,6 +26,7 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi {
     super.onCreate(savedInstanceState)
 
     setContentView(R.layout.sheet_blood_sugar_entry)
+    TheActivity.component.inject(this)
```

Comment:
This need not be done in this PR, but this is an incorrect way of injecting the sheet. Since this is a separate activity, it could cause issues in the event of a process death and restoration since `TheActivity` will not be instantiated when the process is restored if this activity is in the foreground.

We are injecting the rest of the bottom sheets incorrectly, and we should move towards creating separate components for them as soon as possible.